src_Core/RISCY_OOO/procs/lib/Decode.bsv

```diff
@@ -487,7 +487,13 @@ function DecodeResult decode(Instruction inst, Bool cap_mode);
         end
 
         opcJal: begin
-            dInst.iType = J;
+            if (cap_mode) begin
+                dInst.iType = CJAL;
+            end
+            else begin
+                dInst.iType = J;
+            end
+
```

Comment:
Adding a whole CJAL instr type for this is probably overkill, but I thought I'd mirror how it was done with CJALR. It might make sense to clean both of these up in a later commit.